addons/website_slides/models/slide_embed.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from werkzeug import urls
+
+from odoo import _, fields, models
+
+
+class EmbeddedSlide(models.Model):
+    """ Embedding in third party websites. Track view count, generate statistics. """
+    _name = 'slide.embed'
+    _description = 'Embedded Slides View Counter'
+    _rec_name = 'slide_id'
+
+    slide_id = fields.Many2one('slide.slide', string="Presentation", required=True, index=True)
+    url = fields.Char('Third Party Website URL', required=True)
+    count_views = fields.Integer('# Views', default=1)
+
+    def _embed_increment(self, slide_id, url):
```

Comment:
Shoudln't that be on slide model ? As starting from a slide, we search for its embed / create a new one ...